WordPress/src/main/java/org/wordpress/android/util/config/RemoteConfig.kt

```diff
@@ -0,0 +1,42 @@
+package org.wordpress.android.util.config
+
+import com.google.android.gms.tasks.Task
+import com.google.firebase.remoteconfig.FirebaseRemoteConfig
+import com.google.firebase.remoteconfig.FirebaseRemoteConfigSettings.Builder
+import org.wordpress.android.BuildConfig
+import org.wordpress.android.R
+import org.wordpress.android.util.AppLog
+import org.wordpress.android.util.AppLog.T.UTILS
+import javax.inject.Inject
+
+/**
+ * Do not use this class outside of this package. Use [AppConfig] instead
+ */
+class RemoteConfig
+@Inject constructor() {
+    fun refresh() {
+        val firebaseRemoteConfig = FirebaseRemoteConfig.getInstance()
+        val configSettings = Builder()
+                .setMinimumFetchIntervalInSeconds(BuildConfig.REMOTE_CONFIG_FETCH_INTERVAL)
+                .build()
+        firebaseRemoteConfig.setConfigSettingsAsync(configSettings)
+        firebaseRemoteConfig.setDefaultsAsync(R.xml.remote_config_defaults)
+        firebaseRemoteConfig.fetchAndActivate()
```

Comment:
you'll get the value immediately on success but the update is asynchronous so it's likely you tested it on the same screen where you loaded it and before you got the results. If you set an onSuccess listener to the firebase remote config call, you can see that the value is updated when it's done. Also bear in mind that the value gets cached once you request it from the `AppConfig`. The way you can test this is:
- Put your test into an activity that's not Main activity
- Open your app in debug
- Change the field in remote config
- Wait 10 seconds
- Move the app to the background and back to the foreground (this triggers the fetch)
- Go to the sub-activity where you placed your test
- Check that the value is updated

Reply:
thanks for this and additional clarification on slack; I agree that seems safe to cache values in case the app already used the flag to remain consistent 👍!